packages/editor/src/client/atoms/poll/pollExternalVotes.tsx

```diff
@@ -0,0 +1,265 @@
+import {ApolloError} from '@apollo/client'
+import PlusIcon from '@rsuite/icons/legacy/Plus'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import React, {useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Col, IconButton, InputNumber, Message, Modal, Row, Table, toaster} from 'rsuite'
+import FormControl from 'rsuite/FormControl'
+
+import {
+  FullPoll,
+  PollAnswerWithVoteCount,
+  PollExternalVote,
+  PollExternalVoteSource,
+  useCreatePollExternalVoteSourceMutation,
+  useDeletePollExternalVoteSourceMutation
+} from '../../api'
+
+interface PollExternalVotesProps {
+  poll?: FullPoll
+  onExternalSourceCreated(): Promise<void>
+  onExternalSourceDeleted(): Promise<void>
+  onExternalSourceChange(externalVoteSources: PollExternalVoteSource[]): void
+  savePoll(): Promise<void>
+}
+
+export function PollExternalVotes({
+  poll,
+  onExternalSourceCreated,
+  onExternalSourceDeleted,
+  onExternalSourceChange,
+  savePoll
```

Comment:
Is this to fix that when the poll is not saved that we don't override it when we create a new external vote?
I think this is a bit hacky and is an unexpected user flow.

We probably should use a lazy query on the parent so the poll doesn't get updated all the time and thus overriding the data.

Reply:
changed.